GiniVision/Classes/Core/GiniConfiguration.swift

```diff
@@ -584,6 +594,28 @@ import UIKit
      */
     @objc public var reviewTextBottomColor = UIColor.white
     
+    // MARK: Multipage options
+    
+    /**
+     Sets the color of the pages container and toolbar
```

Comment:
I know you're working on an updated integration guide / documentation. It might make sense to expand on these comments a little more. They don't provide more information than the variable names. Which view is that container and where and how does the toolbar show.

Reply:
That is a good idea, but would take a lot of time. I added a lot of comments in the next PR, but I can add more in another PR. I created the issue for that.